zaza/openstack/charm_tests/vault/utils.py

```diff
@@ -280,6 +288,35 @@ def ensure_secret_backend(client):
         pass
 
 
+def wait_for_ready_all(clients):
+    """Return once vault is ready (for all clients).
+
+    Raise an error if it's not ready within a reasonable time.
+
+    :param clients: Clients to use to talk to vault
+    :type clients: List[CharmVaultClient]
+    """
+    for client in clients:
+        wait_for_ready(client)
```

Comment:
This code could wait up to 3x200 seconds for the vault to settle; is that really what's wanted?

Reply:
Actually, I was wrong; it's just 200 seconds and then "bang" with an exception, so it's *probably* okay.